Script/build_packages.py

```diff
@@ -75,7 +77,7 @@ def refresh_staging():
     return staging_path
 
 def extract_into(destdir, zippath):
-    with zipfile.ZipFile(zippath, "r") as z:
+    with contextlib.closing(zipfile.ZipFile(zippath, "r")) as z:
```

Comment:
oh right! I was only looking at the diff. What systems were you hoping to support that were stuck with python 2.6?